docs/src/app/AppRoutes.jsx

```diff
@@ -128,6 +130,7 @@ const AppRoutes = (
       <Route path="time-picker" component={TimePickerPage} />
       <Route path="toggle" component={TogglePage} />
       <Route path="toolbar" component={ToolbarPage} />
+      <Route path="stepper" component={StepperPage} />
```

Comment:
Could you put it after `Snackbar` Please? :grin: 

Also you need to add this to the LeftNav. the only way to access this now is through manually writing the path. 

Add a line after this one: https://github.com/callemall/material-ui/blob/master/docs/src/app/components/AppLeftNav.jsx#L171